{"uid":"7dcc61bd27b88b9","name":"test_network_cut","fullName":"tests.integration.high_availability.test_self_healing#test_network_cut","historyId":"bb2aa2dc0f3a078ccd9996c1af51d19e","time":{"start":1721094098675,"stop":1721094121480,"duration":22805},"description":"Completely cut and restore network.","descriptionHtml":"<p>Completely cut and restore network.</p>\n","status":"broken","statusMessage":"subprocess.CalledProcessError: Command '['lxc', 'config', 'device', 'add', 'juju-06bb51-1', 'eth0', 'none']' returned non-zero exit status 1.","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 433, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 112, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 155, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 108, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 433, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 112, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 155, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 108, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-operator/mysql-operator/tests/integration/high_availability/test_self_healing.py\", line 173, in test_network_cut\n    cut_network_from_unit(primary_hostname)\n  File \"/home/runner/work/mysql-operator/mysql-operator/tests/integration/helpers.py\", line 392, in cut_network_from_unit\n    subprocess.check_call(cut_network_command.split())\n  File \"/usr/lib/python3.10/subprocess.py\", line 369, in check_call\n    raise CalledProcessError(retcode, cmd)\nsubprocess.CalledProcessError: Command '['lxc', 'config', 'device', 'add', 'juju-06bb51-1', 'eth0', 'none']' returned non-zero exit status 1.","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1721093594798,"stop":1721093594798,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1721093594799,"stop":1721093594799,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1721093594799,"stop":1721093594931,"duration":132},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1721094096630,"stop":1721094096630,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1721094096630,"stop":1721094098674,"duration":2044},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Completely cut and restore network.","status":"broken","statusMessage":"subprocess.CalledProcessError: Command '['lxc', 'config', 'device', 'add', 'juju-06bb51-1', 'eth0', 'none']' returned non-zero exit status 1.","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 433, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 112, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 155, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 108, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 433, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 112, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 155, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 108, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-operator/mysql-operator/tests/integration/high_availability/test_self_healing.py\", line 173, in test_network_cut\n    cut_network_from_unit(primary_hostname)\n  File \"/home/runner/work/mysql-operator/mysql-operator/tests/integration/helpers.py\", line 392, in cut_network_from_unit\n    subprocess.check_call(cut_network_command.split())\n  File \"/usr/lib/python3.10/subprocess.py\", line 369, in check_call\n    raise CalledProcessError(retcode, cmd)\nsubprocess.CalledProcessError: Command '['lxc', 'config', 'device', 'add', 'juju-06bb51-1', 'eth0', 'none']' returned non-zero exit status 1.","steps":[],"attachments":[{"uid":"4c7249e4317c6eff","name":"log","source":"4c7249e4317c6eff.txt","type":"text/plain","size":862}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1721094121493,"stop":1721094197849,"duration":76356},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 302, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-operator/mysql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 294, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/work/mysql-operator/mysql-operator/tests/integration/high_availability/conftest.py\", line 32, in continuous_writes\n    await juju_.run_action(application_unit, \"clear-continuous-writes\")\n  File \"/home/runner/work/mysql-operator/mysql-operator/tests/integration/juju_.py\", line 22, in run_action\n    assert result.results.get(\"return-code\") == 0\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1721094197849,"stop":1721094197849,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1721094197857,"stop":1721094198138,"duration":281},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1721094198139,"stop":1721094198139,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1721094198139,"stop":1721094198139,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1721094198139,"stop":1721094198139,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.high_availability"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"fv-az1109-780"},{"name":"thread","value":"7470-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.high_availability.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"ae87cb4fddfb1245","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":53,"unknown":1,"total":56},"items":[{"uid":"7f15ff82f2e902fa","reportUrl":"../2764//#testresult/7f15ff82f2e902fa","status":"passed","time":{"start":1721007970466,"stop":1721008196820,"duration":226354}},{"uid":"256b880379661a01","reportUrl":"../2763//#testresult/256b880379661a01","status":"passed","time":{"start":1720921705860,"stop":1720921933777,"duration":227917}},{"uid":"8895e86589552b59","reportUrl":"../2762//#testresult/8895e86589552b59","status":"unknown","time":{}},{"uid":"58590d9ad107a675","reportUrl":"../2758//#testresult/58590d9ad107a675","status":"passed","time":{"start":1720748621839,"stop":1720748848474,"duration":226635}},{"uid":"7dfa0c179607620b","reportUrl":"../2746//#testresult/7dfa0c179607620b","status":"passed","time":{"start":1720662744204,"stop":1720662969588,"duration":225384}},{"uid":"19c845928b4dadf1","reportUrl":"../2740//#testresult/19c845928b4dadf1","status":"passed","time":{"start":1720489441802,"stop":1720489661703,"duration":219901}},{"uid":"d8a4769f3b22f084","reportUrl":"../2737//#testresult/d8a4769f3b22f084","status":"passed","time":{"start":1720403032721,"stop":1720403253328,"duration":220607}},{"uid":"e657ead52a66738a","reportUrl":"../2736//#testresult/e657ead52a66738a","status":"passed","time":{"start":1720317232460,"stop":1720317457556,"duration":225096}},{"uid":"ab7c1263ecc549b","reportUrl":"../2735//#testresult/ab7c1263ecc549b","status":"passed","time":{"start":1720230252697,"stop":1720230462024,"duration":209327}},{"uid":"63a164138f474726","reportUrl":"../2729//#testresult/63a164138f474726","status":"passed","time":{"start":1720143923748,"stop":1720144144114,"duration":220366}},{"uid":"c0cb0aea9e9dec21","reportUrl":"../2723//#testresult/c0cb0aea9e9dec21","status":"passed","time":{"start":1720057291427,"stop":1720057504824,"duration":213397}},{"uid":"78adb0c79fff141f","reportUrl":"../2710//#testresult/78adb0c79fff141f","status":"passed","time":{"start":1719971030224,"stop":1719971238631,"duration":208407}},{"uid":"8483a3c2f18e6376","reportUrl":"../2701//#testresult/8483a3c2f18e6376","status":"passed","time":{"start":1719884310566,"stop":1719884528774,"duration":218208}},{"uid":"9b8e695ff394523a","reportUrl":"../2694//#testresult/9b8e695ff394523a","status":"passed","time":{"start":1719798086680,"stop":1719798302346,"duration":215666}},{"uid":"ffaed4027d7d7106","reportUrl":"../2693//#testresult/ffaed4027d7d7106","status":"passed","time":{"start":1719712057529,"stop":1719712259840,"duration":202311}},{"uid":"c3b4881b65871bb2","reportUrl":"../2692//#testresult/c3b4881b65871bb2","status":"passed","time":{"start":1719625111722,"stop":1719625334827,"duration":223105}},{"uid":"ec1ae2983299e988","reportUrl":"../2672//#testresult/ec1ae2983299e988","status":"broken","statusDetails":"mysql.connector.errors.DatabaseError: 3100 (HY000): Error on observer while running replication hook 'before_commit'.","time":{"start":1719538589175,"stop":1719538774266,"duration":185091}},{"uid":"3d60c6bc20cb6ee","reportUrl":"../2667//#testresult/3d60c6bc20cb6ee","status":"passed","time":{"start":1719452171539,"stop":1719452392716,"duration":221177}},{"uid":"f3be3795ed33f151","reportUrl":"../2660//#testresult/f3be3795ed33f151","status":"passed","time":{"start":1719365865404,"stop":1719366098185,"duration":232781}},{"uid":"d84e371ade87e449","reportUrl":"../2653//#testresult/d84e371ade87e449","status":"passed","time":{"start":1719279428461,"stop":1719279625379,"duration":196918}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"7dcc61bd27b88b9.json","parameterValues":[]}